config/user-libudev.m4

```diff
@@ -0,0 +1,13 @@
+dnl #
+dnl # Check for libudev - needed for vdev auto-online and auto-replace
+dnl #
+AC_DEFUN([ZFS_AC_CONFIG_USER_LIBUDEV], [
+	LIBUDEV=
+
+	AC_CHECK_HEADER([libudev.h], [AC_SUBST([LIBUDEV], ["-ludev"])
+	AC_DEFINE([HAVE_LIBUDEV], 1, [Define if you have libudev])], [])
+
+	AC_CHECK_LIB([udev], [udev_device_get_is_initialized], [
```

Comment:
AC_SEARCH_LIBS is apparently the preferred macro these days.  @ryao pointed this out recently in the recent e853ba351935135487621f79211e87d5f2e39205 commit.